File: String/58. Length of Last Word/LengthOfLastWord.java
class Solution {
    public int lengthOfLastWord(String s) {
        int lengthOflastWord = 0 ;
        for(int i = s.length()-1 ; i >=0 ; i-- ){
            if(lengthOflastWord == 0 && s.charAt(i)==' ')
                continue ; 
                else if(s.charAt(i)==' ')
                    return lengthOflastWord ;
                    else lengthOflastWord++;
        }
        return lengthOflastWord ;
    }
}

File: Arrays & Hashing/Length Of Last Word/LengthOfLastWord.kt
class Solution {
    fun lengthOfLastWord(s: String): Int {
        
    var counter = 0
    var saveTheLast = 0
    for (i in s) {
        if (i == ' ')
            counter = 0
        else {
            counter++
            saveTheLast = counter
        }
    }

    return saveTheLast
        
        
    }
}
